server/apm/apm-server/apm-engine/src/main/java/io/holoinsight/server/apm/engine/elasticsearch/storage/impl/NetworkAddressMappingEsStorage.java

```diff
@@ -30,14 +30,24 @@ protected RestHighLevelClient esClient() {
     return client;
   }
 
+  protected String rangeTimeField() {
+    return NetworkAddressMappingDO.TIME_BUCKET;
+  }
+
   @Override
-  public List<NetworkAddressMappingDO> loadByTime(long timeBucketInMinute) throws IOException {
-    List<NetworkAddressMappingDO> networkAddressMapping = new ArrayList<>();
+  public List<NetworkAddressMappingDO> loadByTime(long startTime) throws IOException {
+    long rangeStartTime = NetworkAddressMappingDO.TIME_BUCKET.equals(rangeTimeField())
```

Comment:
Can these `if-else` be removed if `rangeTimeField()` returns `START_TIME` instead of `TIME_BUCKET` ?